apps/admin-x-settings/package.json

```diff
@@ -49,6 +49,7 @@
     "@testing-library/react": "14.3.1",
     "@tryghost/admin-x-design-system": "0.0.0",
     "@tryghost/admin-x-framework": "0.0.0",
+    "@tryghost/shade": "0.0.0",
```

Comment:
@peterzimon Adding shade as dependency to Admin Settings, as we eventually want to replace the old design system here too. Lmk if you have any concerns here!

Reply:
The only use case for now is the `abbreviateNumber` helper, that we can get from elsewhere too - just thought it's good to introduce shade into settings anyways